.drone.star

```diff
@@ -2789,7 +2806,6 @@ def githubComment(alternateSuiteName):
         ],
         "when": {
             "status": [
-                "success",
```

Comment:
Why are we dropping the `success` message here?

Reply:
we only want to comment when the test fails. Other repos have a similar approach.